package.json

```diff
@@ -103,24 +99,12 @@
     "prettier": "^3.6.2",
     "publint": "^0.2.12",
     "rimraf": "^4.4.1",
-    "run-all": "^1.0.1",
     "simple-git-hooks": "^2.13.1",
     "tsup": "^8.5.0",
-    "tsx": "^3.14.0",
+    "tsx": "^4.20.6",
     "typescript": "^5.9.2",
     "vitest": "^1.6.1"
   },
-  "pnpm": {
-    "peerDependencyRules": {
-      "ignoreMissing": [
-        "@types/download",
-        "download",
-        "express",
-        "safe-compare",
-        "body-parser"
-      ]
-    }
-  },
```

Comment:
pnpm uses pnpm-workspace.yaml now.

Reply:
I just realized you're on Bun now. Do you want a (separate) Bun migration first, or can we get this merged and Bun later?

Reply:
Let's merge this one first!